### PR TITLE
first whack at the problems!

### PR DESCRIPTION
**Pull request by foxholden**

### CODE REVIEW

File: assignments/003-illumina-sequencing-questions/illumina-seq-homework.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
+
+
+* Q7: Choose one from each set of parentheses: the blue part of the remaining strand is at the (3'|5') end and is (the same as|complementary to) the blue oligos on the lawn.  (_from 93.0 to 96.0 seconds in the video_).
+
+The blue part of the remaining strand is at the 3' end and is complementary to the blue oligos on the lawn.
+
+* Q8: The polymerase grows the new daughter strand from 5' to 3'.  Does that accord with your previous answers?  Make sure your answers are consistent with this. (_from 98.0 to 101.0 seconds in the video_).
+
+Yes!
```
Comment (foxholden) [inline assignments/003-illumina-sequencing-questions/illumina-seq-homework.md:47]: Howdy! Just trying out the commenting thing. Also I'm realizing i titled this commit first whack, but really it should say homework finished :)

File: assignments/003-illumina-sequencing-questions/illumina-seq-homework.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+
+* Q2: How does the sequence of the fragment on top compare to the sequence of the fragment on the bottom? (_from 36.5 to 39.0 seconds in the video_).
+
+They are complementary, so the polarity of the top strand is reversed on the bottom strand. At each position a purine and complementary pyrimidine base will be present.
+
+
+* Q3: What is the purpose of the sequencing binding sites? Your answer should include the words "oligonucleotide," "polymerase," "3'-hydroxyl," and "synthesize." (_from 39.0 to 40.5 seconds in the video_).
+
+I think the sequencing binding site is where the sequencing primers bind. This is a part of the adapter separate from the oligo-complementary strand, which binds to an oligonucleotide. The primers are added to the binding site so that a polymerase can add new bases to the free 3'-hydroxyl end of the newly synthesized strand.
```
Comment (eriqande) [inline assignments/003-illumina-sequencing-questions/illumina-seq-homework.md:28]: Great answer!

File: assignments/003-illumina-sequencing-questions/illumina-seq-homework.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+* Q4: Given the direction of travel of the polymerase, is the Original Template anchored to the lawn by its 3' or its 5' end? (_from 78.0 to 81.0 seconds in the video_).
+
+3' end.
+
+* Q5: How does the remaining strand relate to the Original Template? Is the remaining strand anchored by its 3' or 5' end? (_from 85.0 to 88.0 seconds in the video_).
+
+The remain strand is complementary to the original template. It is anchored by its 5' end.
+
+* Q6: Choose one from the parentheses: the purple part on the remaining strand is (the same as || complementary to) the purple lawn oligos.  (_from 88.2 to 92.2 seconds in the video_).
```
Comment (eriqande) [inline assignments/003-illumina-sequencing-questions/illumina-seq-homework.md:38]: No answer here!

File: assignments/003-illumina-sequencing-questions/illumina-seq-homework.md (hunk 1 above)
Comment (eriqande) [reply assignments/003-illumina-sequencing-questions/illumina-seq-homework.md:47]: Cool!  I got your comment!

**Comment by eriqande** [comment]
Great job Holden.  There wasn't an answer to question #6.  Now worries.  we will probably talk about that in class, because a lot of people got a different answer than i did.

**Pull request closed without merging**